x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/slack_simulation.ts

```diff
@@ -7,22 +7,41 @@
 import http from 'http';
 
 export async function initPlugin() {
+  const messages: string[] = [];
+
   return http.createServer((request, response) => {
+    // return the messages that were posted to be remembered
```

Comment:
extended the slack and webhook simulators, to capture the data sent in some requests, and allow it to be retrieved.  We use this in some new fully-cycle function tests that create actions and alerts and track that the payloads generated by the actions are escaped properly.